test/E2ETest/Microsoft.Test.E2E.AspNet.OData/ComplexTypeInheritance/ComplexTypeInheritanceTests.cs

```diff
@@ -342,23 +342,56 @@ public async Task PatchContainingEntity_Matched_DerivedType(string modelMode)
             // Attempt to PATCH nested resource with delta object of the different CLR type
             // will result an error.
             var content = @"
-{
-    'CurrentShape':
+            {
+                'CurrentShape':
+                {
+                    '@odata.type':'#Microsoft.Test.E2E.AspNet.OData.ComplexTypeInheritance.Circle',
+                    'Radius':2,
+                    'Center':{'X':1,'Y':2},
+                    'HasBorder':true
+                },
+                'OptionalShapes': [ ]
+            }";
+
+            StringContent stringContent = new StringContent(content: content, encoding: Encoding.UTF8, mediaType: "application/json");
```

Comment:
Please wrap `IDisposable`s in `using` blocks (for example, `StringContent` and `HttpResponseMessage`)